azure/iac/registry.rego

```diff
@@ -31,7 +46,7 @@ adminUserEnabled_metadata := {
     "Type": "IaC",
     "Product": "AZR",
     "Language": "ARM template",
-    "Policy Title": "ENSURE THAT ADMIN USER IS DISABLED FOR CONTAINER REGISTRY",
+    "Policy Title": "Ensure that admin user is disabled for Container Registry",
```

Comment:
**I am bit confused.** what we are trying to achieve here? Are we making sure 
**the admin user should disabled for container registry?** any reason as it will be disabled by default.
I think it should be **"Ensure admin user is enabled for Container Registry "**
if not entire rule is wrong in terms of policy title. which need to alter.
@farchide @shahinshirvani

Reply:
for now we don't have this policy on master-compliance-test
because we should check existence and check value of 'adminUserEnabled' is set to false which we can not do it by rules of OPA for now
you can check #159 for more information

and about term of policy title, please check https://gsl.dome9.com/D9.AZU.AKS.01.html to make sure about it